package.json

```diff
@@ -18,6 +19,7 @@
   },
   "devDependencies": {
     "@types/node": "^14.0.1",
+    "jsdoc": "^3.6.5",
```

Comment:
Are you sure, that jsdoc is supporting esnext? From my own experience jsdoc had problems with `class`, `async` and other new keywords that es6 brought up

Reply:
Looks like JSDoc added support for EMCA2015. So those keywords should be fully supported.
https://jsdoc.app/

Reply:
Just for context, if @ranguli happens to catch an error due to the used syntax, there's also https://esdoc.org/